lib/temporal/worker/activity_worker.rb

```diff
@@ -95,8 +95,10 @@ def handle_start_activity(task_token, start)
         running_activities.delete(task_token)
       end
 
-      def handle_cancel_activity(task_token, cancel)
-        # TODO: pending implementation
+      def handle_cancel_activity(task_token, _cancel)
+        runner = running_activities[task_token]
+        # TODO: Warn of a missing activity if runner is absent
```

Comment:
Hrmm, I see in Python I warn here too. But in retrospect this may be an error. I cannot think of a time core would send a cancel for an activity that is not in this map, but maybe there'd be a race between when I send a task completion and they send a cancel.

Meh, yeah warning is fine

Reply:
Yeah, I'll add this once we have a logger, but there's definitely a possibility of a race between completing a task and getting a cancellation. Maybe even a debug log is fine since this isn't something that users can act on